src/App.js

```diff
@@ -14,28 +16,71 @@ class App extends React.Component {
       output: "",
       input: "",
       buttonText: "Submit",
-      description: "Description"
-    };
-
+      description: "Description",
+      showExampleForm: false,
+      examples: [],
+    }
     // Bind the event handlers
-    this.handleChange = this.handleChange.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
     this.handleClick = this.handleClick.bind(this);
   }
 
   componentDidMount() {
     // Call API for the UI params
     axios
       .get(UI_PARAMS_API_URL)
-      .then(({ data: { placeholder, button_text, description } }) => {
+      .then(({ data: { placeholder, button_text, description, show_example_form } }) => {
         this.setState({
           input: placeholder,
           buttonText: button_text,
-          description: description
+          description: description,
+          showExampleForm: show_example_form,
         });
+        if (this.state.showExampleForm) {
+          axios
+            .get(EXAMPLE_API_URL)
+            .then(({ data: examples }) => {
+              this.setState({ examples: examples })
+            });
+        }
       });
   }
 
-  handleChange(e) {
+  updateExample(id, body) {
+    axios.put(EXAMPLE_API_URL + "/" + id, body)
+  }
+
+ debouncedUpdateExample = debounce(this.updateExample, 50)
+
+  handleExampleChange = (index, field) => e => {
```

Comment:
Similar comment as the dictionary structure on the server side, since examples have an ID use the ID to reference each example rather than the index